.github/workflows/cibuild.yml

```diff
@@ -18,13 +18,13 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            python-version: 3.7
+            python-version: "3.8"
           - os: ubuntu-latest
-            python-version: 3.8
+            python-version: "3.9"
           - os: ubuntu-latest
-            python-version: 3.9
+            python-version: "3.10"
           - os: macos-latest
-            python-version: 3.8
+            python-version: "3.9"
```

Comment:
GitHub automatically disabled CI builds after several months of inactivity. So I re-enabled that, and bumped these version numbers.